### PR TITLE
PODS-2293 : Scheme variances - TrusteeDetails - [JSON Writes to API1468]

### PR DESCRIPTION
**Pull request by abhijeetardale**
PODS-2293 : Scheme variances - TrusteeDetails - [JSON Writes to API1468]

### CODE REVIEW

File: app/models/PensionsScheme.scala [hunk 1]
```diff
@@ -363,6 +378,33 @@ case class CompanyTrustee(
...
+    company.vatRegistrationNumber,
+    company.payeReference,
+    company.correspondenceAddressDetails,
+    company.correspondenceContactDetails,
+    company.previousAddressDetails.fold(PreviousAddressDetails(isPreviousAddressLast12Month = false))(c => c))
+  )
+}
+
 case class PartnershipTrustee(
                                organizationName: String,
```
Comment (grantrobson) [inline app/models/PensionsScheme.scala]: Spelt wrong - should be organisation

File: app/models/PensionsScheme.scala [hunk 2]
```diff
@@ -245,23 +246,37 @@ case class Individual(
...
+        details.noUtrReason,
+        details.correspondenceAddressDetails,
+        details.correspondenceContactDetails,
+        details.previousAddressDetails.fold(PreviousAddressDetails(isPreviousAddressLast12Month = false))(c => c)))
+  }
+
+  val individualUpdateWrites: Writes[Individual] = (
+    (JsPath \ "personDetails").write[PersonalDetails] and
+      JsPath.write(commonIndividualWrites)
+    ) (getIndividual(_))
```
Comment (grantrobson) [inline app/models/PensionsScheme.scala]: (_) unnecessary

File: app/models/PensionsScheme.scala (hunk 1 above)
Comment (abhijeetardale) [reply app/models/PensionsScheme.scala]: not fixing it as used in fronted as well

File: app/models/PensionsScheme.scala (hunk 2 above)
Comment (abhijeetardale) [reply app/models/PensionsScheme.scala]: not valid

**Pull request merged**